.github/workflows/playwright.yml

```diff
@@ -0,0 +1,53 @@
+# YAML schema for GitHub Actions:
+# https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions
+#
+# Helpful YAML parser to clarify YAML syntax:
+# https://yaml-online-parser.appspot.com/
+
+name: Playwright Tests
+
+on:
+  workflow_call:
+    inputs:
+      trigger-sha:
+        required: true
+        type: string
+
+jobs:
+  test:
+    name: Local Build & Test
+    timeout-minutes: 12
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: Turn up local server
+        run: |
+          cd src/server
+          scripts/setup_local_dev.sh
```

Comment:
One thing I probably forgot to mention: we also want to use the latest UI code to start the server with. To do that,

1. cd into `src/ui`
2. run: `npm install -g @angular/cli` and `npm install`
3. run `./scripts/build_and_update_python_package.sh`. This will build the angular app and copy the files to the proper server directory.

Then you can run `model-explorer...` to start the server with the latest UI.

Reply:
Yes definitely! Can we set up the UI test in a separate PR since it also involves some npm installation and js related debugging?